test/scenarios/linux/tasks/create_windows_instance.yml

```diff
@@ -39,14 +39,14 @@
     - "Dump instance config"
 
 - name: Wait for WinRM
-  wait_for:
+  ansible.builtin.wait_for:
     port: 5986
     host: "{{ item.networkInterfaces.0.accessConfigs.0.natIP if molecule_yml.driver.external_access else item.networkInterfaces.0.networkIP }}"
     delay: 10
   loop: "{{ server.results }}"
 
 - name: Prepare Windows User
-  script: ./files/windows_auth.py --instance {{ item.name }} --zone {{ item.zone | default(molecule_yml.driver.region + '-b') }} --project {{ gcp_project_id }} --username molecule_usr
+  ansible.builtin.script: ./files/windows_auth.py --instance {{ item.name }} --zone {{ item.zone | default(molecule_yml.driver.region + '-b') }} --project {{ gcp_project_id }} --username molecule_usr
```

Comment:
Note that certain 2.9 / 2.10 versions have problems with free arguments when FQCNs are used, so this can potentially break compatibility.

(For example the Ansible version running in https://ansible.softwarefactory-project.io/zuul/status will produce really strange errors when you use `ansible.builtin.command` instead of `command`... Took me a while to figure out that simply reverting to `command` fixed them.)

Reply:
Yeah. Anyway I am afraid that this might have being last attempt to resurrect this plugin. Nobody seems interested to take care of maintenance on it.

That is why I used major version. Not sure if you seen that were invalid jinja in ssh key, one extra curly brace. I wonder how many are using it in production...

Reply:
I did see that, I was surprised that this didn't cause problems before (resp. I was wondering whether this actually was running successfully anywhere).

Reply:
I'm sorry but I don't have the intensive, time (or need for that matter...) to maintain this anymore... I know you requested a couple of reviews from me in the past but clearly I am not able to keep up.

I hope that it can still be useful to some people or that someone will have the motivation to take it into his own hands...

Reply:
@nbttmbrg No worry and thanks for the time you spent on this. We will have to make some changes on how these plugins are maintained (or not) as too many changes are of the same nature and can be done across all plugins at once. I suspect we might put the plugins back in a single repository.